viewer/src/components/Commands.vue

```diff
@@ -95,6 +95,7 @@ import {
   specialActionWarning
 } from "../logic/commands";
 import { researchNames } from "../data/research";
+import { AvailableResearchData } from "@gaia-project/engine/src/available-command";
```

Comment:
This should  be exported from `@gaia-project/engine`, and it can be imported with `import type`.